testing/acceptance_test.py

```diff
@@ -413,6 +413,8 @@ def test_warnings(self, testdir, n, warn_type):
         elif warn_type == "pytest":
             warn_code = """request.config.warn('', 'this is a warning',
                            fslocation=py.path.local())"""
+        elif warn_type == "invalid":
```

Comment:
Thanks @BrandonHoffman, but this test passes for me on current `master`:

```
======================== test session starts ========================
platform win32 -- Python 3.6.3, pytest-3.8.1, py-1.5.2, pluggy-0.7.1 -- x:\pytest-xdist\.env36\scripts\python.exe
cachedir: .pytest_cache
rootdir: x:\pytest-xdist, inifile: tox.ini
plugins: forked-0.2, xdist-1.22.6.dev1+g96af694.d20180818
collected 67 items / 61 deselected

testing/acceptance_test.py::TestTerminalReporting::test_warnings[pytest--n0] PASSED [ 16%]
testing/acceptance_test.py::TestTerminalReporting::test_warnings[pytest--n1] PASSED [ 33%]
testing/acceptance_test.py::TestTerminalReporting::test_warnings[builtin--n0] PASSED [ 50%]
testing/acceptance_test.py::TestTerminalReporting::test_warnings[builtin--n1] PASSED [ 66%]
testing/acceptance_test.py::TestTerminalReporting::test_warnings[invalid--n0] PASSED [ 83%]
testing/acceptance_test.py::TestTerminalReporting::test_warnings[invalid--n1] PASSED [100%]

============== 6 passed, 61 deselected in 1.56 seconds ==============
```

So this is not really reproducing the problem... 😕

Reply:
I've written a test which reproduces the issue, will push shortly. 👍